pyravendb/store/document_session.py

```diff
@@ -19,6 +20,129 @@ def __init__(self, commands, deferred_command_count, entities=None):
         self.deferred_command_count = deferred_command_count
 
 
+class _RefEq:
+    def __init__(self, ref):
+        if isinstance(ref, _RefEq):
+            self.ref = ref.ref
+        self.ref = ref
```

Comment:
I think this should go into `else`? Otherwise you're overwrite what might have been set in line 26?

Reply:
fixed